src/main/kotlin/com/redhat/devtools/intellij/kubernetes/configuration/KubernetesSettings.kt

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.kubernetes.configuration
+
+import com.intellij.openapi.components.*
```

Comment:
we try to avoid *-imports, it would be great if you could replace this individual imports